src/connect/component.jsx

```diff
@@ -0,0 +1,93 @@
+/* @flow */
+import { loadAxo } from "@paypal/connect-loader-component";
+import {
+  getClientID,
+  getClientMetadataID,
+  getUserIDToken,
+  getLogger,
+} from "@paypal/sdk-client/src";
+
+const sendCountMetric = ({
```

Comment:
I'm totally going to copy this for a different component! <- would it make sense to put `sendCountMetric` in a shared util file somewhere?

Reply:
@jshawl yes! @siddy2181 is actually working on that and moving it to `sdk-client`. It's used in SPB a bunch, and we want it here. We copied it over as we navigate the timeline dance, but the intent is to use a shared util of `sendCountMetric` from `sdk-client`.